src/components/astro/AstroClassroomsTable.jsx

```diff
@@ -129,10 +129,7 @@ function AstroClassroomsTable(props) {
                           type="button"
                           className="manager-table__button--as-link"
                           plain={true}
-                          onClick={calculatedCompleteness !== 0 ?
-                            props.showExportModal.bind(null, assignment, classroom) :
-                            null
-                          }
+                          onClick={props.showExportModal.bind(null, assignment, classroom)}
```

Comment:
This change should be reverted before merging. It is here for making debugging easier.